blas/tpls/KokkosBlas2_gemv_tpl_spec_avail.hpp

```diff
@@ -32,6 +32,7 @@ struct gemv_tpl_spec_avail {
                                              LAYOUTY, MEMSPACE)           \
   template <class ExecSpace>                                              \
   struct gemv_tpl_spec_avail<                                             \
+      ExecSpace,                                                          \
```

Comment:
The ``template<> struct GEMV<...>`` specializations in tpl_spec_decl also need the additional template argument

Reply:
We could actually ETI the execution space itself with Kokkos::Serial and Kokkos::OpenMP directly here?

Reply:
@lucbv Yeah, actually I also prefer that. At some point we need to deal with the issue that a serial BLAS will always be used even if ``ExecSpace == OpenMP``, and that will make it easier.